src/Provider/Provider.Web/Views/Part2/AboutEmployer/AboutEmployer.cshtml

```diff
@@ -0,0 +1,29 @@
+﻿@model Esfa.Recruit.Provider.Web.ViewModels.Part2.AboutEmployer.AboutEmployerViewModel
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl govuk-!-margin-bottom-6">@Model.Title</h1>
+        <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+        <partial name="@PartialNames.ValidationSummary" model='new ValidationSummaryViewModel { ModelState = ViewData.ModelState, OrderedFieldNames = Model.OrderedFieldNames }' />
+
+        <partial name="@PartialNames.ReviewSummary" for="Review" />
+
+        <form asp-route="@RouteNames.AboutEmployer_Get">
+            <div esfa-validation-marker-for="EmployerDescription" class="govuk-form-group">
+                <label asp-for="EmployerDescription" class="govuk-label govuk-!-font-weight-bold">Tell us about the organisation</label>
+                <span esfa-validation-message-for="EmployerDescription" class="govuk-error-message"></span>
+                <textarea asp-for="EmployerDescription" class="govuk-textarea character-count" rows="5" data-val-length-max="500"></textarea>
```

Comment:
Shouldn't this number be driven from view model?

Reply:
This is a cut and paste from Employer where we have numerous examples of `data-val-length-max` being defined in the html.  I'm not that concerned by it, its simple enough for a FE developer to maintain.